StripeFinancialConnections/StripeFinancialConnections/Source/Common/HostController.swift

```diff
@@ -42,6 +42,7 @@ class HostController {
     private let api: FinancialConnectionsAPIClient
     private let clientSecret: String
     private let returnURL: String?
+    private let v1AnalyticsClient: STPAnalyticsClientProtocol
```

Comment:
its likely a good idea to include the "v1" to differentiate the two different ones - maybe we should even rename the instance variable for this in `FinancialConnectionsSheet`

what do you think of naming it something like `analyticsClientV1`? the logic is that the important thing to know first is that its an "analytics_client" and only after the context is known, is it needed to differentiate its type ("v1")

so, for example, a developer might be coding and they type "analyticsClient" as a guess to reference an analytics client and it be good for the compiler to suggest/recognize and allow dev to pick the right one (maybe IDE is smart enough to pick up v1AnalyticsClient but the odd prefix of v1 might confuse it)

Reply:
Done!